pkg/resource/deploy/step.go

```diff
@@ -1559,6 +1561,8 @@ func RawPrefix(op display.StepOp) string {
 		return "= "
 	case OpImportReplacement:
 		return "=>"
+	case OpRemovePendingReplace:
+		return "~ "
```

Comment:
We are modifying the resource's state, so I went with `~`, but any other suggestions welcome.